h2o-py/h2o/backend/server.py

```diff
@@ -267,7 +267,6 @@ def _launch_server(self, port, baseport, mmax, mmin, ea, nthreads):
                      "%dM" % (num >> 20) if num == (num >> 20) << 20 else \
                      str(num)
             cmd += [mq + numstr]
-        cmd += ["-verbose:gc", "-XX:+PrintGCDetails", "-XX:+PrintGCTimeStamps"]
```

Comment:
Do we need a replacement for this? Is it okay to just remove?

Reply:
What about adding `-XX:OnOutOfMemoryError="kill -9 %p"`

Reply:
@mmalohlava is this OS-specific? Would it work on Windows?

I am not convinced it is the right thing to do when H2O is started from the client.

Reply:
@mmalohlava We're adding the option to specify custom args in this PR

Reply:
@michalkurka yes needs to be specific per platform - however, simplifies kill of jvm - instead long suffering, easy kill.